### PR TITLE
docker image for armorblox-sdk on demisto

### PR DESCRIPTION
**Pull request by SamhithaTatipalli**
## Status
Ready/In Progress/In Hold (Reason for hold)

## Related Content Pull Request
Related PR: link to the PR at demisto/content

## Related Issues
Related: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

### CODE REVIEW

File: docker/armorblox/Pipfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+armorblox-sdk = "*"
```
Comment (urajat) [inline docker/armorblox/Pipfile:7]: I think we should fix this to a specific version of the package - we are at 0.1.0 I think.

Comment (urajat) [reply docker/armorblox/Pipfile:7]: Although I see that the Pipfile.lock has it fixed at 0.1.0, in which case we might not need to do anything.

File: docker/armorblox/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
+FROM demisto/python3:3.10.4.30607
+
+COPY requirements.txt .
```
Comment (urajat) [inline docker/armorblox/Dockerfile:4]: Where is this requirements.txt file?

File: .github/dependabot.yml [hunk 3]
```diff
@@ -753,10 +753,6 @@ updates:
     directory: /docker/duoadmin3
     schedule:
      interval: daily
-  - package-ecosystem: pip
```
Comment (urajat) [inline .github/dependabot.yml:756]: Was this deliberately removed?

File: docker/armorblox/.gitignore [hunk 4]
```diff
@@ -0,0 +1 @@
+requirements.txt
```
Comment (urajat) [inline docker/armorblox/.gitignore:1]: If we are copying requirements.txt in the Dockerfile, how can we ignore the file?

File: docker/armorblox/Pipfile (hunk 1 above)
Comment (ankita-sharma2019) [reply docker/armorblox/Pipfile:7]: Will let it remain as is

File: .github/dependabot.yml (hunk 3 above)
Comment (ankita-sharma2019) [reply .github/dependabot.yml:756]: No, I believe this is a mistake. Will add these lines back

File: docker/armorblox/Dockerfile (hunk 2 above)
Comment (ankita-sharma2019) [reply docker/armorblox/Dockerfile:4]: this requirement.txt is created by the build process using the `pipfile`. https://github.com/demisto/dockerfiles#advanced

File: docker/armorblox/.gitignore (hunk 4 above)
Comment (ankita-sharma2019) [reply docker/armorblox/.gitignore:1]: This is how all the images directory look like. They support `pipfiles` and the requirement.txt is added from `pipenv` only So, may be they don not want to git maintain it

**Review by urajat — APPROVED** [review]

**Pull request merged**